src/mtev_http1.c

```diff
@@ -996,6 +1001,24 @@ mtev_http1_session_req_consume_read(mtev_http1_session_ctx *ctx,
       }
     }
     else {
+      /* Check if we have enough in this node read the chunk header "$chunk_size\r\n" */
```

Comment:
I don't understand this comment. Perhaps it should read "enough space in this node to read"?

Reply:
fixed